src/main.c

```diff
@@ -299,6 +299,13 @@ static gboolean install_start(int argc, char **argv)
 			goto out_loop;
 		}
 	} else {
+		if (!determine_slot_states(&error)) {
```

Comment:
The commit message doesn't mention that this is copied here for the non-service install case.

Reply:
Not sure if I understand what you mean. It says:

>    An explicit call to determine_slot_states() needs to be re-added for
>    install_start() in the non-service case.